.changeset/angry-scissors-argue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/coin-tron": minor
+---
+
+On tron, API has a hard limit on the number of transaction to retrieve. The API returns the oldest set of transactions it can find according to the minHeight parameter.
```

Comment:
```suggestion
On tron, API has a hard limit on the number of operations to retrieve. The API returns the oldest set of operations it can find according to the minHeight parameter.
```